src/github.rs

```diff
@@ -146,8 +147,65 @@ impl GitHubApi {
         }
         Ok(result)
     }
+
+    /// Get all teams for the rust-lang org
+    pub(crate) fn teams(&self) -> Result<Vec<GitHubTeam>, Error> {
+        Ok(self
+            .prepare(true, Method::GET, "orgs/rust-lang/teams?per_page=100")?
+            .send()?
+            .error_for_status()?
+            .json()?)
+    }
+
+    /// Get all team members for the team with the given id
+    pub(crate) fn team_members(&self, name: &str) -> Result<Vec<GitHubMember>, Error> {
+        let mut members = Vec::new();
+        let mut page_num = 1;
+        loop {
+            let page: Vec<GitHubMember> = self
+                .prepare(
+                    true,
+                    Method::GET,
+                    &format!(
+                        "orgs/rust-lang/teams/{}/members?per_page=100&page={}",
+                        name, page_num
+                    ),
+                )?
+                .send()?
+                .error_for_status()?
+                .json()?;
+            let len = page.len();
+            members.extend(page);
+            if len < 100 {
```

Comment:
FWIW, the "proper" way to do this is via the Link header (https://docs.github.com/en/rest/guides/traversing-with-pagination), but that's a pain so this seems OK for now.

Reply:
Weirdly, these API endpoints don't seem to be returning `Link` headers:

```bash 
$ curl -I https://api.github.com/orgs/rust-lang/teams/compiler/members\?per_page\=100\&page\=1 -H "Authorization: token $MYTOKEN" | grep -i link
access-control-expose-headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
```

A quick search shows several people noting that many endpoints don't return that header. 🤷‍♀️